src/c++/vdo/base/io-submitter.c

```diff
@@ -234,31 +231,40 @@ static struct vio *get_mergeable_locked(struct int_map *map, struct vio *vio,
 		return NULL;
 
 	if (back_merge) {
+<<<<<<< HEAD
 		return (get_bio_sector(vio_merge->bios_merged.tail) == merge_sector ?
 			vio_merge : NULL);
 	}
 
 	return (get_bio_sector(vio_merge->bios_merged.head) == merge_sector ?
+=======
+		return (vio_merge->bios_merged.tail->bi_iter.bi_sector == merge_sector ?
+			vio_merge : NULL);
+	}
+
+	return (vio_merge->bios_merged.head->bi_iter.bi_sector == merge_sector ?
+>>>>>>> 1f46120 (dm vdo io-submitter: remove get_bio_sector)
 		vio_merge : NULL);
 }
```

Comment:
Conflict marker here. I think you can keep the second block.

Reply:
Ah I see you fixed this, but in the second patch instead of the first. Should be in the first patch, though.